wp-content/themes/pawar2018/page-sidebar.php

```diff
@@ -15,8 +15,7 @@
       <div class="row align-center">
         <div class="small-12 medium-12 columns">
           <h1 class="section-title"><?php echo the_field('inner_section_title'); ?></h1>
-          <?php the_field('ngp_van_js_embed'); ?>
-          <?php the_field('ngp_van_div_embed'); ?>
+          <?php the_content(); ?>
```

Comment:
Yep! I thought that we needed the separate text field to initialize the script, but it turns out we don't. It might've changed in a recent WP update? ¯\_(ツ)_/¯